src/browser/squirrel-update.coffee

```diff
@@ -21,6 +22,9 @@ fileKeyPath = 'HKCU\\Software\\Classes\\*\\shell\\Atom'
 directoryKeyPath = 'HKCU\\Software\\Classes\\directory\\shell\\Atom'
 backgroundKeyPath = 'HKCU\\Software\\Classes\\directory\\background\\shell\\Atom'
 environmentKeyPath = 'HKCU\\Environment'
+openWithKeyPath = 'HKCU\\Software\\Classes\\Atom'
+filetypeKeyPath = (fileExtension) -> "HKCU\\Software\\Classes\\.#{fileExtension}\\OpenWithProgIds"
+openWithFileExtensions = require path.join(binFolder, 'win32-open-file-extensions.json')
```

Comment:
> Will move it if desired

Yeah, I do think we should move it to somewhere in `src/`

> If so, should it just go in the root of src or create a new subdirectory?

I think that is a good place to put it :+1: